src/management/api/proxy/backend/healthcheck/healthcheck.controller.ts

```diff
@@ -92,25 +98,32 @@ class ApiHealthCheckController {
         from: this.query.from,
         to: this.query.to
       },
-      {notify: false});
+      { notify: false });
 
     this.ApiService.apiHealthLogs(this.api.id, this.query).then((logs) => {
       this.logs = logs.data;
     });
-    this.ApiService.apiHealthLogs(this.api.id, _.assign({transition: true}, this.query)).then((logs) => {
+    this.ApiService.apiHealthLogs(this.api.id, _.assign({ transition: true }, this.query)).then((logs) => {
       this.transitionLogs = logs.data;
     });
 
-    let from: any;
-    let to: any;
-    from = this.query.from || moment().subtract(1, 'days');
-    to = this.query.to || moment();
-    let interval = Math.floor((to - from) / 24);
-    let promises = [
-      this.ApiService.apiHealthAverage(this.api.id, {from: from, to: to,
-        interval: interval, type: 'RESPONSE_TIME'}),
-      this.ApiService.apiHealthAverage(this.api.id, {from: from, to: to,
-        interval: interval, type: 'AVAILABILITY'})
+    const from = this.query.from || moment().subtract(1, 'days').valueOf();
+    const to = this.query.to || moment().valueOf();
```

Comment:
I added some `.valueOf()` otherwise we are dealing with Moment instance and I guess it is not what we want as we are doing some maths with them on the next line.